DependencyInjection/Configuration.php

```diff
@@ -106,12 +106,24 @@ public function getConfigTreeBuilder()
                     ->addDefaultsIfNotSet()
                     ->canBeEnabled()
                     ->children()
+                        ->arrayNode('defaults')
```

Comment:
this is the real purpose of this PR: we can now configure defaults for all sitemap configurations in one place, and override them specifically.